source/extensions/filters/http/buffer/buffer_filter.h

```diff
@@ -53,20 +55,21 @@ class BufferFilter : public StreamDecoderFilter {
   void onDestroy() override;
 
   // Http::StreamDecoderFilter
-  FilterHeadersStatus decodeHeaders(HeaderMap& headers, bool end_stream) override;
-  FilterDataStatus decodeData(Buffer::Instance& data, bool end_stream) override;
-  FilterTrailersStatus decodeTrailers(HeaderMap& trailers) override;
-  void setDecoderFilterCallbacks(StreamDecoderFilterCallbacks& callbacks) override;
+  Http::FilterHeadersStatus decodeHeaders(Http::HeaderMap& headers, bool end_stream) override;
+  Http::FilterDataStatus decodeData(Buffer::Instance& data, bool end_stream) override;
+  Http::FilterTrailersStatus decodeTrailers(Http::HeaderMap& trailers) override;
+  void setDecoderFilterCallbacks(Http::StreamDecoderFilterCallbacks& callbacks) override;
 
 private:
   void onRequestTimeout();
   void resetInternalState();
 
   BufferFilterConfigConstSharedPtr config_;
-  StreamDecoderFilterCallbacks* callbacks_{};
+  Http::StreamDecoderFilterCallbacks* callbacks_{};
   Event::TimerPtr request_timeout_;
   bool stream_destroyed_{};
 };
-
-} // Http
+}
```

Comment:
nit: missing closing namespace BufferFilter comment

Reply:
OK thanks. I will fix in the next one. Can you approve?

Reply:
sounds good 🚢